deploy/deploy_ancillaryRH7.yml

```diff
@@ -0,0 +1,142 @@
+    custom_repo: true
+  vars_files:
+    - ~/RHACCOUNT
+  become: yes
+  become_method: sudo
+  pre_tasks:
+    - debug:
+        msg: "Variables: _uid = {{ _uid }}, _pwd = **********"
+#       msg: "Variables: _uid = {{ _uid }}, _pwd = {{ _pwd }}"
+    - name: register RHEL7.5
+      command: "/usr/bin/subscription-manager register --username {{ _uid }} --password {{ _pwd }} --auto-attach"
+      when: _uid != "" and _pwd != ""
+#     command: "/usr/bin/subscription-manager register --username <name> --password <password> --auto-attach"
+    - name: register RHEL7 repo
+      command: "/usr/bin/subscription-manager repos --enable=rhel-7-server-rpms --enable=rhel-7-server-extras-rpms --enable=rhel-7-server-optional-rpms"
+    - name: register RHEL7 RHSCL Repo
+      shell: "/usr/bin/subscription-manager repos --enable=rhel-server-rhscl-7-rpms"
+    - name: register RHEL7 DevTools
+      shell: "/usr/bin/subscription-manager repos --enable=rhel-7-server-devtools-rpms"
+    - name: list RHEL enabled repos
+      shell: "/usr/bin/subscription-manager repos --list-enabled"
+      register: enabled_repos
+    - debug:
+        msg: "RHEL Enabled Repos: {{ enabled_repos }}" 
+# tasks:
+    - name: download epel
+      shell:  wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -O /tmp/epel-release-latest-7.noarch.rpm   
+    - name: Install epel
+      shell:  yum -y localinstall /tmp/epel*rpm
+
+    - name: install X Window System
+      shell: yum -y groupinstall "X Window System"
+
+    - name: install XFCE
+      shell: yum -y groupinstall "xfce"
+
+    - name: install Evince
+      shell: yum -y install evince
+
+    - name: Install python-pip
+      shell: yum install -y python27 python-pip
+    - name: install gdown
+      shell: pip install gdown  
+#     yum:
+#       name: python27-python-pip
+#       state: latest
+    - name: Install python 3.6 and pip
+      shell: yum install -y python36 python36-pip
+
+#   - name: enable scl
+#     become: yes
+#     become_method: sudo
+#     shell: scl enable python27 bash
+#   - name: install gdown
+#     shell: /opt/rh/python27/root/usr/bin/pip install gdown
+
+#     Already secure copy to destination host
+#   - name: use gdown to download and build local repo
+#     shell: gdown https://drive.google.com/uc?id=1qaZH9_SrGyjHjWT3P2WtvZAhEkP9YG1Z -O /tmp/ancilRH7_repo.tar.gz
+
+#     shell: gdown https://drive.google.com/uc?id=139Z0MfyT0WVWIzP_uwM2KxPCrfYs_vUV
+#   - name: Install pip
+#     easy_install:
+#       name: pip
+#       state: latest
+#    - name: Build custom local repo
+#      shell: wget https://drive.google.com/open?id=1qaZH9_SrGyjHjWT3P2WtvZAhEkP9YG1Z /tmp
+    - name: make localrepo directory
+      shell: mkdir -p /opt/ancillary_local
+    - name: make OpenOffice directory
+      shell: mkdir -p /opt/ancillary_local/OO416
+    - name: make Sublime3 directory
+      shell: mkdir -p /opt/ancillary_local/Sublime3
+    - name: make Sublime3 install directory
```

Comment:
@lel99999 What are these folders for?

Reply:
folders are for custom local repo to stage installation files like /artifacts, will update to use /artifacts instead